scrapper.py

```diff
@@ -1,6 +1,11 @@
 """
 Crawler implementation
 """
+import requests
+from bs4 import BeautifulSoup
+from time import sleep
+import random
+from constants import PROJECT_ROOT, ASSETS_PATH, CRAWLER_CONFIG_PATH
```

Comment:
First of all, fix code style - as it is the first check here. Remove trailing whitespaces , correct import order, variable naming and so on.